tests/testthat/test-build-articles.R

```diff
@@ -4,3 +4,40 @@ test_that("can recognise intro variants", {
   expect_true(article_is_intro("pack-age", "pack.age"))
   expect_true(article_is_intro("articles/pack-age", "pack.age"))
 })
+
+test_that("can build article that uses html_vignette", {
```

Comment:
canNOT?

Reply:
Nope definitely can 😄 This test ensures that `build_article()` special cases `html_vignette()` even though it doesn't support `theme`